### PR TITLE
Remove unnecessary gitignores

### LINKED ISSUE #2668: Update gitignore (s)
**Notes:**
Requestor: Amber

**AC:**
Get ride of ignore lines that not longer exist:
- HMTL
- React
- Angular
- Base Gatsby

### LINKED ISSUE #2672: Remove makefile, we don’t need it
**Notes:**
Requestor: Amber

**AC:**
- Remove makefile

### PR DESCRIPTION
**Pull request by RVMendoza**
## What does this PR do?
Refactor gitignore and delete makefile 

### Associated Issue 
Fixes #2668
Fixes #2672

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,41 +1,7 @@
-demo
 dist
 node_modules
-npm-debug.log
-*.swp
-*.swo
-.DS_Store
```
Comment (afebbraro) [inline .gitignore]: We need .DS_Store, it gets added by macOS when new files are added unfortunately

File: angular/.gitignore [hunk 2]
```diff
@@ -1,48 +1,5 @@
...
-/connect.lock
 /coverage
-/libpeerconnection.log
-npm-debug.log
-yarn-error.log
-testem.log
-/typings
-
-# System Files
-.DS_Store
```
Comment (afebbraro) [inline angular/.gitignore]: we want to keep .DS_Store, it gets added by OS sometimes

File: angular/.gitignore [hunk 3]
```diff
@@ -1,48 +1,5 @@
...
-# IDE - VSCode
-.vscode/*
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
-.history/*
-
-# misc
-/.sass-cache
```
Comment (afebbraro) [inline angular/.gitignore]: we dont want to commit the sass-caches, should keep

File: angular/.gitignore [hunk 4]
```diff
@@ -1,48 +1,5 @@
...
-/bazel-out
-
-# dependencies
 /node_modules
-
-# profiling files
-chrome-profiler-events*.json
-speed-measure-plugin*.json
-
-# IDEs and editors
```
Comment (afebbraro) [inline angular/.gitignore]: we dont want to commit editor stuff, so should keep these

File: html/.gitignore [hunk 5]
```diff
@@ -1,4 +1,3 @@
 dist/
 es5/
 storybook-static/
-.nyc_output/
```
Comment (afebbraro) [inline html/.gitignore]: do we not have a generated nyc_output dir anymore?

Comment (RVMendoza) [reply html/.gitignore]: we do not

File: angular/.gitignore (hunk 4 above)
Comment (RVMendoza) [reply angular/.gitignore]: so I'm wondering why we would need this when it's never existed in other gitignores.

Comment (afebbraro) [reply angular/.gitignore]: @RCopeland and others have personal editor configs setup that we don't want to accidentally commit. I would like you to research why this wasn't a problem for us before.

File: html/.gitignore (hunk 5 above)
Comment (afebbraro) [reply html/.gitignore]: I just checked and we do still have one. If you run `npm run coverage` you can see it. So this cannot be removed. 
Screen Shot 2020-01-29 at 9 00 30 AM

**Pull request merged**